packages/appstore/src/types/params.types.ts

```diff
@@ -1,5 +1,5 @@
 export type TRoute = {
-    component?: React.ReactNode;
+    component?: () => void;
```

Comment:
This shouldn't be changed. Please find out why `src/constants/routes-config.ts` is throwing an error.

Reply:
Since react components are functions that return something we need to change the type to a function type which return reactnode (as matin said to support a wider range of types).
Making this small change fixes the issue that we have in our build process.

Reply:
cc: @matin-binary